code/controllers/master.dm

```diff
@@ -617,3 +617,14 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 		processing = config.base_mc_tick_rate
 	else if(client_count > config.high_pop_mc_mode_amount)
 		processing = config.high_pop_mc_tick_rate
+
+/datum/controller/master/proc/formatcpu()
+	switch(world.cpu)
+		if(0 to 80) // 0-80 = green
+			. = "<font color='#32a852'>[world.cpu]</font>"
```

Comment:
can just return it here right? No need to set `.`

Reply:
All the other MC related state handlers use `. =`, so I am keeping it with the format
```dm
/datum/controller/subsystem/proc/state_letter()
	switch(state)
		if(SS_RUNNING)
			. = "R"
		if(SS_QUEUED)
			. = "Q"
		if(SS_PAUSED, SS_PAUSING)
			. = "P"
		if(SS_SLEEPING)
			. = "S"
		if(SS_IDLE)
			. = "  "

/datum/controller/subsystem/proc/state_colour()
	switch(state)
		if(SS_RUNNING) // If its actively processing, colour it green
			. = "<font color='#32a852'>"
		if(SS_QUEUED) // If its in the running queue, but delayed, colour it orange
			. = "<font color='#fcba03'>"
		if(SS_PAUSED, SS_PAUSING) // If its being paused due to lag, colour it red
			. = "<font color='#eb4034'>"
		if(SS_SLEEPING) // If fire() slept, colour it blue
			. = "<font color='#4287f5'>"
		if(SS_IDLE) // Leave it default if the SS is idle
			. = "<font>"
```